pkg/promlib/models/query.types.json

```diff
@@ -69,6 +69,13 @@
               "type": "string",
               "x-enum-description": {}
             },
+            "groupByKeys": {
+              "description": "Group By parameters to apply to aggregate expressions in the query",
```

Comment:
This is used to generate schemas. right? Perhaps we should make an comment about it being experimental.

Reply:
I'm not sure what this json is used for, but experimental is also true of AdhocFilters and Scopes props ... so going to not worry about this right now :-)